server/games/game.py

```diff
@@ -70,6 +72,7 @@ def __init__(
         enforce_rating_range: bool = False,
         max_players: int = 12,
         setup_timeout: int = 60,
+        hosted_at: datetime = datetime_now()
```

Comment:
This isn't going to work because argument default are only created once at the time the function is defined. I.e. this default datetime will always be the time when the server is booted up. You have to do it like I wrote earlier, where you use `None` as the default and call the function later. 

It's the same reason why you have to be really careful with using mutable values as defaults:
```python
def foo(data=[]):
    data.append(10)
    return data
```
Just paste that into the interpreter and call it a few times. You'll see what I mean.

This one will work as expected:
```python3
def foo(data=None):
    data = data or []
    data.append(10)
    return data
```